MainApp/lib/app/routes/api.js

```diff
@@ -50,12 +47,12 @@ export default function (router) {
                             // TODO: Redirect to Thank you for already playing the game page.
                             res.send({success: false, message: "Username already exists"});
                         } else {
-                            res.send({success: true, userid: username, message: "Trial Information saved"});
+                            res.send({success: true, userid: trialinfo.username, message: "Trial Information saved"});
```

Comment:
@yousuf031991 .. Yeah we need to. Consider following scenario-  Game is in progress and the user closes the browser before finishing the game. The gamepage URL looks like '/gamePage/<username>'. So in order to resume from the game page the next time, we need the username.